setup.cfg

```diff
@@ -28,7 +28,7 @@ classifiers =
 include_package_data = True
 python_requires = >=3.9
 install_requires =
-  spacy>=3.1.0,<3.4.0
+  spacy>=3.0.0
```

Comment:
Is changing from 3.1 to 3.0 going to cause any issues?

Reply:
from what I've tested on 3.9 and 3.11, no; although, I don't remember changing the version from 3.1 to 3.0. I'll change this to the original 3.1